web/CPU_README.md

```diff
@@ -1,6 +1,6 @@
-  
-# CPU Version
 
+# CPU Version
+Instructions below show how to install and set up WEB OTE for Linux. The same steps are required for Windows and MacOS.
```

Comment:
was it tested on such OSes?

Reply:
No, but there is not any reason why this tool could not work on such OSes. As far as we do not support it officially, I can remove this mention

Reply:
I would remove things that have not been validated somehow at least